bitnami/redis/templates/configmap-scripts.yaml

```diff
@@ -29,11 +29,15 @@ data:
         if [[ ! -f /data/redisboot.lock ]]; then
           export REDIS_REPLICATION_MODE="master"
         else
+          {{- if .Values.usePassword }}
           if is_boolean_yes "$REDIS_TLS_ENABLED"; then
             sentinel_info_command="redis-cli -a $REDIS_PASSWORD -h {{ template "redis.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} -p {{ .Values.sentinel.port }} --tls --cert ${REDIS_TLS_CERT_FILE} --key ${REDIS_TLS_KEY_FILE} --cacert ${REDIS_TLS_CA_FILE} info"
```

Comment:
I think this could be done on the same line without having to copy the command twice

```
redis-cli {{-if .Values.password}} -a $REDIS_PASSWORD {{- end }} -h ...

Reply:
That's true. I did it because I didn't want to change both lines of each if, but it really doesn't make much sense to add it to the command when `$REDIS_TLS_ENABLED=true`.